bin/install-test-framework.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -ev
 if [ "${TEST_FRAMEWORK}" = "ember-mocha" ]; then
-  yarn remove ember-qunit 2>/dev/null && echo "n" | ember install ember-mocha || true
+  yarn remove ember-qunit 2>/dev/null && echo "n" | ember install ember-mocha || echo "n" | ember g ember-mocha || true
```

Comment:
I'm not sure about the why it fails to generate `ember-mocha` blueprints via `ember install ember-mocha` now.

This "fix" work and force to generate them in case of fails, but feel free to share if you have an idea about the why / a better fix